collectors/proc.plugin/proc_spl_kstat_zfs.c

```diff
@@ -372,6 +372,10 @@ int do_proc_spl_kstat_zfs_pool_state(int update_every, usec_t dt)
                         pool->unavail = 1;
                     } else {
                         disable_zfs_pool_state(pool);
+
+                        char *c = strchr(state, '\n');
```

Comment:
neat: how about doing it before we if/elseif ?

Reply:
I don't like the idea. Actually, `strchr` is a loop. I don't see the point of doing it on every file read.